### PR TITLE
MM-18620 - Marketplace icons white background 

### PR DESCRIPTION
**Pull request by marianunez**
#### Summary

Added white background to non-default plugin icons in order they can properly render in dark themes when they have transparent backgrounds. 

Screen Shot 2019-09-19 at 9 46 21 PM

#### Ticket Link
Fixes [18620](https://mattermost.atlassian.net/browse/MM-18620)

### CODE REVIEW

**Comment by marianunez** [comment]
@asaadmahmood these changes does not include the `border-radius` or `padding` like you suggested. For the padding, I was not able to not make the icons tiny. Let me know if you have any suggestions on this. 

Regarding the `border-radius`, discussing with @lieut-data we had some concerns if it would cut into the required area around the logo as some specs require.

**Comment by asaadmahmood** [comment]
@lieut-data Yes, I think we can do that, we can have the marketplace live separately because of the issues it may have with the icons, and just use the default mattermost theme styling instead.

That way, we would also not need the padding and the border-radius stuff.

File: components/widgets/icons/close_icon.jsx [hunk 1]
```diff
@@ -7,7 +7,10 @@ import {FormattedMessage} from 'react-intl';
 export default class CloseIcon extends React.PureComponent {
     render() {
         return (
-            <span {...this.props}>
+            <button
+                className='style--none close-x'
```
Comment (marianunez) [inline components/widgets/icons/close_icon.jsx]: Should we have these changes for the CloseIcon and modal css on a separate PR? This is not related to marketplace specifically and affects the modals in general.

Comment (marianunez) [reply components/widgets/icons/close_icon.jsx]: This change was causing issues updating the snapshots and blocking the CI build for this PR. I've moved these changes to this PR https://github.com/mattermost/mattermost-webapp/pull/3835
